spec/factories/answer_ratings.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :answer_rating do
+    
```

Comment:
where are attributes here? if you don't use this factory, just remove it.

Reply:
Completed.